atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -269,4 +257,28 @@ private String getErrorMessage(String tableName, List<CharacterLimitType> kvsExc
                         + "If running only against a different KVS, set the ignoreTableNameLength flag.",
                 tableName, StringUtils.join(kvsExceeded, ", "));
     }
+
+    static void assertGeneratedFiles(Path actualDir, Path expectedDir) throws IOException {
+        if (Boolean.getBoolean("recreate")) {
+            FileUtils.deleteDirectory(expectedDir.toFile());
+            FileUtils.copyDirectory(actualDir.toFile(), expectedDir.toFile());
+        }
+
+        files(actualDir).forEach(path -> {
```

Comment:
might be missing something: not sure you want to run this if recreate was specified?